Runtime/CourseController/StandaloneCourseController.cs

```diff
@@ -480,7 +486,13 @@ private void SetupModePicker()
             modePicker.value = RuntimeConfigurator.Configuration.Modes.CurrentModeIndex;
             
             // If there is only one option, the dropdown is currently disabled.
-            modePicker.enabled = availableModes.Count > 1;
+            if (availableModes.Count > 1 == false)
```

Comment:
```suggestion
            if (availableModes.Count < 2)
```
Seems more readable to me. But I am also fine without the change.